### DISCUSSION TITLE
Gp book reading

### DISCUSSION THREAD

**Discussion by xEllesi** [GriefPrevention/GriefPrevention#1336] [Ideas]
Hi, i want to ask if its possible add an option to read-books inside claims on lecterns
thanks <3

**Comment by Jikoo** [comment]
You should be able to read books in lecterns with accesstrust.

**Reply by xEllesi** [reply]
with that the owner needs to do accesstrust to al players in server

**Reply by Jikoo** [reply]
That is correct. You can create a small subclaim around the lectern if that's a problem.

**Reply by xEllesi** [reply]
but players didnt want to add all the players on the server, they just want to they to come, read and 'use' their claim

**Reply by Jikoo** [reply]
Not seeing the issue with using `/accesstrust public`, could you clarify how that's problematic?

**Reply by xEllesi** [reply]
cuz with that command they can open another things that they didnt want to acess

**Reply by Jikoo** [reply]
> You can create a small subclaim around the lectern if that's a problem.

You can easily work around allowing access to other things.

Goshdarn mobile, entered that early.

**Reply by xEllesi** [reply]
eh ok thx

**Reply by RoboMWM** [reply]
That's the current solution. I'm not sure if this is something we should just make public for anyone though by default considering that most of the use cases are for public use. I'm not in any rush to implement that, but I'd be open to PRs for it.
